docker-compose.yml

```
@@ -48,7 +48,16 @@ services:
     depends_on:
       - twoopstracker_app
 
+  pesacheck_meedan_bridge:
+    image: codeforafrica/pesacheck_meedan_bridge:latest
+    command: ["tail", "-f", "/dev/null"]
+    volumes:
+      - ./pesacheck-data:/app/database
```

```suggestion
      - pesacheck-data:/app/database
```
I'm assuming this isn't an actual folder on project directory.